app/src/test/java/net/aiscope/gdd_app/presentation/MetadataMapperTest.kt

```diff
@@ -0,0 +1,103 @@
+package net.aiscope.gdd_app.presentation
+
+import android.content.Context
+import com.nhaarman.mockito_kotlin.whenever
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import net.aiscope.gdd_app.CoroutineTestRule
+import net.aiscope.gdd_app.R
+import net.aiscope.gdd_app.model.MalariaSpecies
+import net.aiscope.gdd_app.model.MalariaStage
+import net.aiscope.gdd_app.model.SmearType
+import net.aiscope.gdd_app.ui.metadata.MetadataMapper
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+
+@ExperimentalCoroutinesApi
+@RunWith(MockitoJUnitRunner::class)
+class MetadataMapperTest {
+    @get:Rule
+    var coroutinesTestRule = CoroutineTestRule()
```

Comment:
The `MetadataMapper` doesn't make use of coroutines, so we don't need this test rule.

Reply:
Thanks. I removed this rule in changeset c0f5c9692349070bbd81db509eab0ed21928085e.